en/docs/install-and-setup/configuring-sms-otp-authenticator.md

```diff
@@ -1,4 +1,5 @@
-This document provides step by step instructions to set up SMS OTP configurations.
+This document provides step by step instructions to set up SMS OTP configurations. For more information,
+see [Consumer Data Standards - Authentication flows](https://consumerdatastandardsaustralia.github.io/standards/#authentication-flows).
```

Comment:
Let's mention the requirement, why we are setting this up and for that check the spec.